ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-chain.yaml

```diff
@@ -5,8 +5,6 @@ chain:
   - ref: ipi-conf-telemetry
   - ref: ipi-conf-aws
   - ref: ipi-conf-aws-byo-ipv4-pool-public
-  - ref: ipi-conf-aws-user-min-permissions
```

Comment:
the mini-perm provision steps were removed from `ipi-conf-aws`, instead, adding `- chain: aws-provision-iam-user-minimal-permission` to each workflow before install step, the reason is installer generates mini-perm policy based on install-config, so the policy will be generated at the last minute before install step.

Reply:
Cool